std/range/package.d

```diff
@@ -3319,6 +3318,7 @@ if (isBidirectionalRange!R)
     assert("hello world".take(6).drop(3).equal("lo "));
 }
 
+///
```

Comment:
This is fine. It ensures that the following unittest is an example for `dropBack`

Reply:
Is there anything else I should do?